modules/textual_inversion/textual_inversion.py

```diff
@@ -168,7 +167,7 @@ def load_from_file(self, path, filename):
         if 'string_to_param' in data:
             param_dict = data['string_to_param']
             if hasattr(param_dict, '_parameters'):
-                param_dict = getattr(param_dict, '_parameters')  # fix for torch 1.12.1 loading saved file from torch 1.11
+                param_dict = param_dict._parameters  # fix for torch 1.12.1 loading saved file from torch 1.11

```

Comment:
This hasattr/getattr dance simplifies to
```suggestion
            param_dict = getattr(param_dict, '_parameters', param_dict)  # fix for torch 1.12.1 loading saved file from torch 1.11
```